README.rst

```diff
@@ -42,11 +42,13 @@ Or write a benchmark script ``bench.py``:
     import perf
 
     runner = perf.Runner()
-    runner.timeit("sorted(list(range(1000)), key=lambda x: x)",
+    runner.timeit(name="sort a sored list",
                   stmt="sorted(s, key=f)",
                   setup="f = lambda x: x; s = list(range(1000))")
 
-Run the script using (result written into ``bench.json``)::
+See `the API docs`_ for full details on the ``timeit`` function and the
+``Runner`` class. To Run the script and dump the results into a file named
```

Comment:
"To Run" should be written "To run", no?

Reply:
d'oh!

fixed.